src/design-guidelines/color-system/examples/contrast-color-palette.scss

```diff
@@ -1,12 +1,6 @@
 @use '../../../style/functions';
```

Comment:
The default snackbar colour seems to have changed. At first I thought it was because I was running with the system in dark mode and Kompendium in forced light mode, and that the snackbar had always been white in true light mode (because I never use that, so I wouldn't know), but when I switched the system to light mode there was no difference.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212075845-19993e04-4b42-4378-a68d-f8dcbce027f5.png">

Not a show-stopper in my opinion. I'm noting it here so we can fix it eventually. I would prefer if we _don't_ fix it now, because depending on the fix, I might have to review all the components again… 😅

Reply:
Yes, this kind of became a logical result of all the changes. 
I write more in this issue: https://github.com/Lundalogik/lime-elements/issues/2027